.changeset/twenty-masks-punch.md

```diff
@@ -0,0 +1,104 @@
+---
+"webdriver-image-comparison": minor
+"@wdio/visual-service": minor
+"@wdio/ocr-service": patch
+---
+
+### @wdio/visual-service
+
+#### 🚀 New Features
+
+**Added Reporting output**
+You now have the option to export the compare results into a JSON report file. By enabling the module option `createJsonReportFiles: true`, each image that is compared will create a report stored in the `actual` folder, next to each `actual` image result.
+
+The output will look like this:
+
+```json
+{
+    "check methods": {
```

Comment:
mhm 🤔 I wonder if this is an ideal JSON key, given that if someone imports it into a JS file and tries to access the data they would have to use brackets for it, e.g. `result['check methods'].` instead of e.g. `result.data` or whatever

Reply:
Updated the structure, curious what you think about it now